book/src/intro.md

```diff
@@ -60,7 +60,8 @@ weigh before using them, especially in regards to performance and scalability.
 [ud]: https://www.unrealengine.com/
 [jme]: http://jmonkeyengine.org/
 
-We think that basing the Amethyst engine on good and modern principles will allow us to make an open source game engine that can actually be more performant than those engines.
+We think that basing the Amethyst engine on good and modern principles will allow us to make an open source game engine
```

Comment:
Drop the we, let's leave the opinions and jump into facts territory ;)

Reply:
@jojolepro did you read my review comment on this?

Reply:
yeah

Reply:
Ok, so which alteration of this sentence would you prefer, yours or mine?

Reply:
I'm not suggesting a sentence. I was suggesting to have the first sentence be a fact instead of an opinion, to have a strong credible start